File: 54.spiral-matrix/test_one.py
import time
from spiral_matrix_leetcode_simulation import *
# from spiral_matrix_leetcode_layer_by_layer import *
# from spiral_matrix_my import *
from data import *

if __name__ == "__main__":
    solution = Solution()

    num_tests = 100000
    tic = time.clock()
    
    for _ in range(num_tests):
        for input in inputs:
            solution.spiralOrder(input)

    toc = time.clock()

    print(round((toc-tic) * 1000), "ms with", num_tests, "tests")

File: 54.spiral-matrix/TestOne.java
class TestOne {

    public static void main(String[] args) {
        SpiralMatrixLeetcode solution = new SpiralMatrixLeetcode();;
        Data data = new Data();;
     
        Integer num_tests = 100000;
        long tic = System.currentTimeMillis();

        for (int i = 0; i < num_tests; i++)
            for (int[][] input : data.inputs) 
                solution.spiralOrder(input);

        long toc = System.currentTimeMillis();

        System.out.println(toc-tic + " ms with " + num_tests + " tests");
    }
}
